src/main/java/sg/ncl/MainController.java

```diff
@@ -144,39 +144,121 @@ public String tools() {
         return "tools";
     }
 
-    @RequestMapping(value="/futureplan/download", method=RequestMethod.GET)
-    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException, IOException {
+    @RequestMapping("/createaccount")
+    public String createaccount() {
+        return "createaccount";
+    }
+
+    @RequestMapping("/createexperiment")
+    public String createexperiment() {
+        return "createexperiment";
+    }
+
+    @RequestMapping("/applyteam")
+    public String applyteam() {
+        return "applyteam";
+    }
+
+    @RequestMapping("/jointeam")
+    public String jointeam() {
+        return "jointeam";
+    }
+
+//    @RequestMapping("/resource2")
+//    public String resource2() {
+//        return "resource2";
+//    }
+
+//    @RequestMapping("/admin2")
+//    public String admin2() {
+//        return "admin2";
+//    }
+
+
+    @RequestMapping("/tutorials")
+    public String tutorials() {
+        return "tutorials";
+    }
+
+
+//    @RequestMapping("/dataresource")
+//    public String dataresource() {
+//        return "dataresource";
+//    }
+
+//    @RequestMapping(value="/futureplan/download", method=RequestMethod.GET)
+//    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException, IOException {
+//        InputStream stream = null;
+//        response.setContentType("application/pdf");
+//        try {
+//            stream = getClass().getClassLoader().getResourceAsStream("downloads/future_plan.pdf");
+//            response.setContentType("application/force-download");
+//            response.setHeader("Content-Disposition", "attachment; filename=future_plan.pdf");
+//            IOUtils.copy(stream, response.getOutputStream());
+//            response.flushBuffer();
+//        } catch (Exception ex) {
+//            logger.info("Error writing file to output stream.");
+//            throw new FuturePlanDownloadException("IOError writing file to output stream");
+//        } finally {
+//            if (stream != null) {
+//                stream.close();
+//            }
+//        }
+//    }
+
+    @RequestMapping(value="/orderform/download", method=RequestMethod.GET)
+    public void OrderForm_v1Download(HttpServletResponse response) throws OrderFormDownloadException, IOException {
+        InputStream stream = null;
+        response.setContentType("application/pdf");
+        try {
+            stream = getClass().getClassLoader().getResourceAsStream("downloads/order_form.pdf");
+            response.setContentType("application/force-download");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "application/force-download" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)